roles/web/tasks/setup_virtualenv.yml

```diff
@@ -8,8 +8,11 @@
   command: virtualenv -p {{ virtualenv_python_version }} {{ virtualenv_path }}
            creates={{ virtualenv_path }}/bin/activate
 
-- name: Ensure gunicorn is installed
-  pip: virtualenv={{ virtualenv_path }} name=gunicorn
+- name: Ensure gunicorn and pycurl are installed in the virtualenv
+  pip: virtualenv={{ virtualenv_path }} name={{ item }}
+  with_items:
+    - gunicorn
+    - pycurl
```

Comment:
`pycurl` is required to perform Django database migrations. Please see the commit message for 	33a1a5a.